docs/crafted-emacs.org

```diff
@@ -144,6 +150,23 @@ for the code examples.
     (straight-use-package 'corfu)
   #+end_src
 
+  You can find a full example to bootstrap straight in the examples directory:
+
+  #+begin_src emacs-lisp
+    (find-file "../examples/crafted-early-init-straight.el")
+  #+end_src
+
+  Assuming you copied it to your ~custom-modules~ folder, you can load it in
```

Comment:
I might consider this an alternate approach.  The example file could be copied (or merged) with/as the user's `early-init.el` file